packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1372,6 +1311,68 @@ class SemanticsNode extends AbstractNode with DiagnosticableTreeMixin {
     _dirty = false;
   }
 
+  /// Builds a new list made of [_children] sorted in semantic traversal order.
+  List<SemanticsNode> _childrenInTraversalOrder() {
+    TextDirection inheritedTextDirection = textDirection;
+    SemanticsNode ancestor = parent;
+    while (inheritedTextDirection == null && ancestor != null) {
+      inheritedTextDirection = ancestor.textDirection;
+      ancestor = ancestor.parent;
+    }
+
+    List<SemanticsNode> childrenInDefaultOrder = _childrenInDefaultOrder(_children, inheritedTextDirection);
+    if (inheritedTextDirection != null) {
+      childrenInDefaultOrder = _childrenInDefaultOrder(_children, inheritedTextDirection);
```

Comment:
Why call `_childrenInDefaultOrder` twice in this case?

Reply:
My bad, the first one is unnecessary.